script/sign_binaries.py

```diff
@@ -12,10 +12,8 @@
 
 def get_sign_cmd(file):
   # https://docs.microsoft.com/en-us/dotnet/framework/tools/signtool-exe
-  sdk_dir = os.path.normpath(os.environ.get(
-      'WINDOWSSDKDIR',
-      'C:\\Program Files (x86)\\Windows Kits\\10'))
-  cmd = "{}\\bin\\x64\\signtool.exe {}".format(sdk_dir, signtool_args)
+  # signtool should be in the path if it was set up correctly by gn through src/build/vs_toolchain.py
+  cmd = 'signtool ' + format(signtool_args)

```

Comment:
actually shouldn't this be `'signtool {}'.format(signtool_args)`? I guess both ways work?

Reply:
yeah that's not intentional, will fix this.

Reply:
@bridiver mind taking a look at this again?